Clicker/Views/PollBuilder/MCPollBuilderView.swift

```diff
@@ -181,7 +185,16 @@ extension MCPollBuilderView: PollBuilderMCOptionSectionControllerDelegate {
         let newMCOptionModel = PollBuilderMCOptionModel(type: .newOption(option: "", index: mcOptionModels.count - 1))
         mcOptionModels.insert(newMCOptionModel, at: mcOptionModels.count - 1)
         updateTotalOptions()
-        adapter.reloadData(completion: nil)
+        pollBuilderDelegate.ignoreNextKeyboardHiding()
+        adapter.reloadData { _ in
+            let index = IndexPath(item: 0, section: self.mcOptionModels.count - 2)
```

Comment:
what does `self.mcOptionModels.count - 2` refer to?

Reply:
This is because the last cell (at `mcOptionModels.count - 1`) is the add options button, so we want to select the one before that, the last cell for typing an option.

I'll leave a comment in the code to clarify.